src/Tasks/Microsoft.NET.Build.Tasks/ProjectContext.cs

```diff
@@ -12,6 +12,9 @@ namespace Microsoft.NET.Build.Tasks
 {
     internal class ProjectContext
     {
+        private const string NetCorePlatformLibrary = "Microsoft.NETCore.App";
+        private const string AspNetPlatformLibrary = "Microsoft.AspNetCore.App";
```

Comment:
I think this can also be Microsoft.AspNetCore.*All* on 2.x.

Reply:
Maybe consider any platform library that is not base Microsoft.NETCore.App to depend on Microsoft.NETCore.App?

Reply:
[You are correct](https://github.com/aspnet/Universe/blob/da48de7d7580ce316784dd3e250786ffa7017c25/src/Packages/Microsoft.AspNetCore.All/build/netcoreapp3.0/Microsoft.AspNetCore.All.targets#L10).

Perhaps a check for "not empty and not `Microsoft.NETCore.App`" would be sufficient?

Reply:
Yes, that sounds good.

Reply:
Pushed.